lib/src/main/kotlin/com/swmansion/starknet/data/types/StringWrapper.kt

```diff
@@ -0,0 +1,8 @@
+package com.swmansion.starknet.data.types
+
+import com.swmansion.starknet.data.serializers.StringWrapperSerializer
+import kotlinx.serialization.Serializable
+
+@JvmInline
+@Serializable(with = StringWrapperSerializer::class)
+value class StringWrapper(val value: String) : HttpBatchRequestType
```

Comment:
the same question as per IntWrapper value class

Reply:
Retrieving string value from `StarknetString` will be the same as from `StarknetInt` - we have to access `value` property.